monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx

```diff
@@ -11,11 +11,17 @@ type Props = { url: string, filename: string, variant?: string }
 const LOG_FILE_NOT_FOUND_ERROR = "The server returned a 404 (NOT FOUND) response: " +
                                  "The requested log files do not exist."
 
-export const AgentLogDownloadButton = ({ url, filename, variant = 'primary' }: Props) => {
+export const AgentLogDownloadButton = ({ url, agentIds, filename, variant = 'primary' }: Props) => {
```

Comment:
Update line 9 with `agentIds`.

Reply:
That is going to be fixed in the other PR.

Reply:
https://github.com/guardicore/monkey/pull/2704